packages/inventory/src/components/PreConfirmationError/index.tsx

```diff
@@ -42,11 +23,25 @@ interface PreConfirmationErrorProps {
  */
 const PreConfirmationError = (props: PreConfirmationErrorProps) => {
   const { errorObj } = props;
+  useTranslation();
```

Comment:
Is there way to avoid having this in every single component - maybe by somehow putting it into App.tsx or index.tsx?

Reply:
🤔 could work, will look into this

Reply:
Nice, looks like you managed to pull this off.  Do you think it is worth investigating making it even more decoupled?  Maybe implemented as a provider, or something like that?  Primarily thinking of how to make it super easy to use in Reveal, for example.  Bonus points for other projects like GRP, Akuko, etc. However if going down this road is too much work I'm fine with putting down your thoughts in an issue.

Reply:
yeah, the biggest hurdle overall has been decoupling, not having to place constraints on other users of theses packages, I propose we continue the conversation and trials from another issue.